### ISSUE TITLE
Failing to build cosmic-settings vendored

### ISSUE THREAD

**Issue by ryanabx** [pop-os/cosmic-settings#740]
```
error: failed to sync

Caused by:
  found duplicate version of package `smithay-client-toolkit v0.19.2` vendored from two sources:

  	source 1: registry `crates-io`
  	source 2: https://github.com/Smithay/client-toolkit#618a8764
```

This only started happening recently

**Comment by git-f0x** [comment]
Hopefully I didn't mess up something, since I did a `cargo update` with the latest commit, and I did notice the fixed version of sctk being added to Cargo.lock.

**Comment by ryanabx** [comment]
> Hopefully I didn't mess up something, since I did a `cargo update` with the latest commit, and I did notice the fixed version of sctk replacing the non-versioned one in Cargo.lock. Edit: I think I did mess it up. :( Edit2: The same thing happens when I run `cargo update` or `cargo update libcosmic` on other repos.

Could be a change in libcosmic caused it, and it just propagated to here

**Comment by git-f0x** [comment]
Yeah, the same thing seems to be happening in the Accessibility applet PR.

**Comment by Drakulix** [comment]
Ping @wash2

**Comment by wash2** [comment]
Maybe some dependency pulled it in. I'll look through the dependencies to see which one and hopefully get it fixed soon. A patch could probably be a quick fix though.

**Comment by wash2** [comment]
Oh, its the panel update. I'll check why it is using a git dep and replace it if possible.

**Comment by wash2** [comment]
ahh it's because panel needs the same calloop version for sctk as smithay, and the updated calloop is unreleased for sctk. I think I was waiting for a 1.0 release. I think we should just use a patch in settings for now and in the applets too. If that is not alright, I can do an sctk release.
